lookout/style/format/feature_extractor.py

```diff
@@ -307,22 +310,23 @@ def _parse_vnodes(self, files: Iterable[File], lines: Optional[List[List[int]]]
         parsed_files = []
         index_labels = not self.labels_to_class_sequences
         for i, file in enumerate(files):
-            contents = file.content.decode("utf-8", "replace")
+            path = file.path
             uast = file.uast
+            file = AnnotatedData.from_file(file)
             try:
-                file_vnodes, file_parents = self._parse_file(contents, uast, file.path)
+                # Should be self._parse_file(content, file)
```

Comment:
wdym "should be"? TODO or FIXME?

Reply:
I mean when I finish refactoring. I address this comment in the next PR
Here I decided to keep `_parse_file` signature the same because I do not want to change tests now.